src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+
+    <article class="card" data-qa="card">
+      <a href="#" class="card__img-link">
+        <img
+          class="img"
+          src="images/imac.jpeg"
+          alt="product-1"
+        >
+      </a>
+      <h1 class="card__header">
```

Comment:
it's better to use `h2` since we use `h1` as page title
```suggestion
      <h2 class="card__header">
```